src/java.desktop/share/classes/javax/swing/JTable.java

```diff
@@ -5481,6 +5481,42 @@ public Component getTableCellRendererComponent(JTable table, Object value,
 
             return this;
         }
+
+        @Override
+        public AccessibleContext getAccessibleContext() {
+            if (accessibleContext == null) {
+                accessibleContext = new AccessibleBooleanRenderer();
+            }
+            return accessibleContext;
+        }
+
+        protected class AccessibleBooleanRenderer
```

Comment:
I think this class, `AccessibleBooleanRenderer` as well as `setCellToDoActionOn` method can have the default access. They're inside package-private class `BooleanRenderer`, I see no reason why these should have higher access level.